custom/ewsghana/balance.py

```diff
@@ -106,6 +108,10 @@ def validate_web_users(self, date):
                     object_type='webuser'
                 ).delete()
 
+        migration_stats = EWSMigrationStats.objects.get(domain=self.domain)
+        migration_stats.web_users_count = len(unique_usernames)
+        migration_stats.save()
+
```

Comment:
Since we balance the web user count here, I think we should get rid of this line: https://github.com/dimagi/commcare-hq/pull/8813/files#diff-39c1ba856e2df8f52eef6a76748f75d7R241